webapp/src/main/resources/locale/portlet/Portlets_en.properties

```diff
@@ -1112,3 +1112,16 @@ categoryInput.restrictedCategoryLink=Restricted access. You can't manage this ca
 categoryInput.cancel=Cancel
 categoryInput.update=Update
 categorySuggester.searchCategories=Search for a category
+
```

Comment:
Please split the I18N file to add a dedicated file for this portlet to not load those keys in all pages while it should be used in simple storage only.

Reply:
Done